src/main/java/org/spongepowered/api/entity/living/Living.java

```diff
@@ -294,23 +294,4 @@
      * @param invisible Whether this entity is invisible or not
      */
     void setInvisible(boolean invisible);
-
-    /**
-     * Gets whether this living entity is invisible to the specific
-     * {@link Player} entity.
-     *
-     * @param player The other player to check
-     * @return Whether this entity is invisible to the given player
-     */
-    boolean isInvisibleTo(Player player);
```

Comment:
Yes, but that doesn't mean remove it from the `master` branch, which is what you're doing here.